### DISCUSSION TITLE
Some symbols/characters do not render correctly

### DISCUSSION THREAD

**Discussion by gnakicta** [LazyVim/LazyVim#3583] [Q&A]
- Did a fresh install yesterday on macOS with all of the dependencies and nerd fonts
- Most of the icons render correctly, but there are some symbols that render either as a black box or as a question mark, as shown in the screenshot
- Tried multiple nerd fonts, changing terminal emulators from iTerm to Kitty or Wezterm, but got the same issue

![gh-issue](https://github.com/LazyVim/LazyVim/assets/80105956/0a7e69da-a9ea-4ea0-ac4a-8dc44ba03cc9)

What could I be missing?

**Comment by dpetka2001** [comment]
The question marks in Markdown are because of your fonts. See this [comment](https://github.com/lukas-reineke/headlines.nvim/issues/41#issuecomment-1556334775) from `headlines.nvim` repo. The lualine component I'm guessing it's because of your colorscheme and you might be able to fix it by adding some padding maybe (not 100%sure, might be totally wrong)??

**Reply by gnakicta** [reply]
Thanks! I resolved the issue with headlines from that comment. The second issue is indeed because of the colorscheme. Figuring out how to configure padding without messing the rest of the lualine config lazyvim does.

**Reply by gnakicta** [reply]
referencing https://github.com/LazyVim/LazyVim/discussions/3459 for the same issue as above
